readme.md

```diff
@@ -1,12 +1,35 @@
-![Tridactyl logo](src/static/logo/Tridactyl_200px.png)
-
-# Tridactyl [![Build Status](https://travis-ci.org/tridactyl/tridactyl.svg?branch=master)](https://travis-ci.org/tridactyl/tridactyl) [![Matrix Chat][matrix-badge]][matrix-link] [![Gitter Chat][gitter-badge]][gitter-link]
-
-Replace Firefox's default control mechanism with one modelled on the one true editor, Vim.
-
-![Gigantic GIF showing Tridactyl in action](doc/AMO_screenshots/trishowcase.gif)
-
-## Installing
+<h1 align="center">
+<br>
+<img src="src/static/logo/Tridactyl_200px.png" alt="Tridactyl Logo" width="200">
+<br>
+Tridactyl
+<br>
+</h1>
+
+<h4 align="center">Replace Firefox's default control mechanism with one modelled on the one true editor, Vim. </h4>
+
+<p align="center">
+<a href="https://travis-ci.org/tridactyl/tridactyl"><img src="https://travis-ci.org/tridactyl/tridactyl.svg?branch=master" alt="Build Status"></a>
+<a href="https://gitter.im/tridactyl/Lobby"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Join Gitter Chat"></a>
```

Comment:
We need to fix the Matrix badge rather than remove it as it's our "official" channel. Gitter is just a bridge, and it sometimes doesn't work : )